src/selector.js

```diff
@@ -0,0 +1,35 @@
+const parser = require('postcss-selector-parser');
+
+const caseSensitivePseudoClassesLists = [
+  ':active',
+  ':hover',
+  ':focus',
+  ':focus-within',
+  ':visited',
+];
+
+const caseSensitivePseudoClassesSet = new Set(caseSensitivePseudoClassesLists);
+
+function getParsed(selectors, callback) {
+  return parser(callback).processSync(selectors);
+}
+
+export const transformer = (str) =>
+  getParsed(str, (sel) => {
+    sel.walkTags((selNode) => {
+      selNode.value = selNode.value.toLowerCase();
```

Comment:
Let's use https://github.com/sindresorhus/html-tags and lowercase only known

Reply:
`walkTags` will traverse over all the html tags right. I guess this package can be added to `postcss-selector-parser` . What do you think?

Reply:
Just add tests and check it, i am not manual :smile:

Reply:
@evilebottnawi I added more tests https://github.com/anikethsaha/postcss-lowercase-text/pull/3/commits/a2065c5f65531d851309dc5ad4d516edab578689 and here https://github.com/anikethsaha/postcss-lowercase-text/pull/3/commits/652cd43349fad5df918c1ac0ee32707e9e9aadf8